src/pyFAI/test/test_fiber_integrator.py

```diff
@@ -312,16 +312,16 @@ def test_integrate1d_defaults(self):
                                                                npt_oop=100, npt_ip=500,
                                                                vertical_integration=True)
 
-        self.assertEqual(abs(res1d_vertical_ref.radial - res1d_vertical.radial).max(), 0)
-        self.assertEqual(abs(res1d_vertical_ref.intensity - res1d_vertical.intensity).max(), 0)
+        numpy.allclose(res1d_vertical_ref.radial, res1d_vertical.radial)
```

Comment:
You are note making a test (assert) to validate the result is correct ... Here the test always passes, the result of `numpy.allclose` can just be False.